.github/workflows/deploy.yml

```diff
@@ -0,0 +1,78 @@
+name: Deployment
+on:
+  push:
+    branches: 'version2'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Get current date
+      id: date
+      run: echo "today=$(date +'%Y-%m-%d-%H-%M')" >> $GITHUB_OUTPUT
+    - name: 'Authenticate to Google Cloud - building'
+      id: 'auth_build'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        workload_identity_provider: 'projects/855475113448/locations/global/workloadIdentityPools/eto-github/providers/eto-github'
+        service_account: 'eto-artifact-registry-github@gcp-cset-projects.iam.gserviceaccount.com'
+        token_format: 'access_token'
+    - name: 'Log into Artifact Registry'
+      uses: 'docker/login-action@v2'
+      with:
+        registry: us-east1-docker.pkg.dev
+        username: 'oauth2accesstoken'
+        password: '${{ steps.auth_build.outputs.access_token }}'
+    - name: Setup node
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+    - name: 'Cache node_modules'
+      uses: actions/cache@v3
+      with:
+        path: gui/node_modules
+        key: js-${{ hashFiles('gui/package-lock.json') }}
+    - name: 'Get node_modules'
+      working-directory: gui
+      run: |
+        npm run artifactregistry-login
+        npm install
+    - name: 'Build assets'
+      working-directory: gui
+      run: |
+        npm install -g gatsby-cli
+        gatsby build
+    - name: 'Authenticate to Google Cloud - deployment'
+      id: 'auth_deploy'
+      uses: 'google-github-actions/auth@v1'
+      with:
+        workload_identity_provider: 'projects/855475113448/locations/global/workloadIdentityPools/eto-github/providers/eto-github'
+        service_account: 'eto-cd-github@gcp-cset-projects.iam.gserviceaccount.com'
+    - name: 'Set up Cloud SDK'
+      uses: 'google-github-actions/setup-gcloud@v1'
+    - name: 'Copy built assets to cloud'
+      id: 'gcloud'
+      working-directory: gui
```

Comment:
Should this be `web/gui-v2`?

Reply:
Good catch - yes, the working directories should be `gui-v2`